src/js/menu/menu-button.js

```diff
@@ -76,9 +76,24 @@ class MenuButton extends Button {
    */
   createItems(){}
 
+  createEl() {
+    return super.createEl('div', {
+      className: this.buildCSSClass()
+    });
+  }
+
   /** @inheritDoc */
   buildCSSClass() {
-    return `${this.className} vjs-menu-button ${super.buildCSSClass()}`;
+    return `vjs-menu-button ${super.buildCSSClass()}`;
+  }
+
+  controlText(text) {
+    if (!text) return super.controlText();
+
+    this.controlText_ = text;
+    this.el().children[0].children[0].innerHTML = this.controlText_;
```

Comment:
This whole thing can go away since we could just override `this.controlTextEl_` in `MenuButton` if we needed to change the location.